backend/tests/api/user_test.go

```diff
@@ -38,3 +41,93 @@ func TestGetAllUsersWorks(t *testing.T) {
 	// assert that the user returned from the database is the same as the user returned from the API
 	assert.Equal(dbUser, respUser)
 }
+
+func TestGetUserHappyPath(t *testing.T) {
+	// initialize the test
+	app, assert := InitTest(t)
```

Comment:
see this new [helper](https://github.com/GenerateNU/sac/blob/main/backend/tests/api/helpers.go#L179) I made to make this testing easier / less boilerplate for you guys! here is an [example test](https://github.com/GenerateNU/sac/blob/main/backend/tests/api/category_test.go#L13) for inspiration